setup.py

```diff
@@ -34,7 +34,7 @@
 
 setup(
     name = 'OpenFisca-Core',
-    version = '24.6.4',
+    version = '24.7.0',
```

Comment:
Isn't it an API breaking change?

Reply:
Can you clarify? I expect no reusers are *relying* on the fact that the API server crashes if your model contains the wrong type of parameter, so I would classify this as "adds functionality in a backward-compatible manner".

Reply:
Hi @Morendil 

I can't quickly determine with confidence, just by reading the code, if there's any change to the json schema (a test would be welcome).

If there's only additions to the schema, I agree that this "adds functionality in a backward-compatible manner".

But I don't know, that's why I'm asking.